app/assets/stylesheets/shared/_postcode-lookup.scss

```diff
@@ -2,4 +2,8 @@
   padding: govuk-spacing(3);
   margin: govuk-spacing(6) 0;
   background: govuk-colour("light-grey");
+
+  .govuk-hint .gem-c-govspeak {
```

Comment:
It could be worth a comment to explain why this override is in place - I'm guessing it's because you render something in Govspeak which makes the text black, and you want to revert it to make the hint colour grey?